packages/scripts/config/tsconfig/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "isolatedModules": false,
     "jsx": "react",
     "lib": ["dom", "es2015", "es2016", "es2017"],
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
what impact does this have?

Reply:
Gives us new language features but code output is the same. From what I read it is only the dynamic import support.